topics/transcriptomics/tutorials/scrna-intro/slides.html

```diff
@@ -0,0 +1,1228 @@
+   * Cell Type by Fluorescent Labelling
+     *  Cell Surface Markers (CDs)
+     *  Fluorescent Labelling
+]
+]
+
+.pull-left[
+.reduce90[
+* Isolate a cell into its own sequencing environment
+]
+]
+
+???
+- Flow cytometry floats cells in a shallow liquid bath and streams them along a narrow channel, just narrow for one cell to pass through.
+- Cells can be screened by a variety of properties this way, such as by their light scatter properties, and from fluorescent cell labelling.
+- Cells can be tagged and isolated in this manner.
+
+---
+
+# Capture / Sorting: Size and Type
+
+.pull-right[
+  ![](../../images/scrna-intro/opticssystem.png)
+]
+
+
+.pull-left[
+*Optical Scatter*
+  * Ratio of Cell Size:Wavelength
+  * If Cell Size < Laser Wavelength (~400nm)
+    * Low intensity and high inconsistency scatter
+
+  * Measured in terms of:
+     * Forward Scatter (FSC)
+     * Side Scatter (SSC)
+
+]
+
+???
+- Optical scatter properties can be used to probe size and consistency of the cell, where cells with a smaller size than the laser wavelength yield lower intensities and more inconsistent scatter patterns.
+- There are two main types of optical scatter: Forward scatter, and Side scatter.
+
+---
+
+# Capture / Sorting: Size and Type
+
+.pull-left[
+.reduce90[
+*Forward Scatter (FSC)*
+  * Measures along the path of the laser
+  * FSC intensity proportional to diameter of cell
+  * Good for distinguishing between immune cells
+]
+]
+
+.image-75[.pull-right[![](../../images/scrna-intro/FlowJo_Layouts__01-Mar-2017.jpg)]]
+
+
+???
+Forward scatter is aligned with the main laser and measures the diameter of cell, which is ideal for distinguishing different cells by their size profiles, such as monocytes which are typically larger than lymphocytes as seen on the X-axis of the example image.
+
+--
+
+.pull-left[
+.reduce90[
+<br />
+
+*Side Scatter (SSC)*
+  * Measures 90° to laser, along path of cells
+  * Much weaker intensities than FSC
+  * Refraction/reflection proportional to granularity of cell
+]
+]
+
+.image-75[.pull-right[![](../../images/scrna-intro/Granulocytes_vs_Monocytes_scatter.jpg)]]
+
+
+???
+Side scatter is perpendicular to the main laser, and measures the granularity of the cell, ideal for distinguishing cells with less defined internal structures, such as the granulocytes on the Y-axis of the example image.
+
+---
+
+# Capture / Sorting: FACS
+
+.pull-left[
+![](../../images/scrna-intro/CD8vsCD3.png)
+.footnote[.reduce70[Image from BD Biosciences]]
+]
+
+.pull-right[
+.reduce90[
+*Fluorescence-Activated Cell Sorting (FACS)*
+  * Cell surface markers
+    * Fluorescent Markers for each cell
+  * Positive and Negative
+    * Whether cell activated for that CD or not.
+  * Plot different CD markers against each other
+    * Isolate cell populations
+  * Can set gating thresholds to isolate analysis to enriched subset of cells
+
+]
+]
+
+???
+- Cells can also be gated and characterised by their cell surface markers via FACS.
```

Comment:
Should this be spoken as F.A.C.S, or like "fax"? I think it will do the former now (right @hexylena?), if you want the latter, maybe write it in lowercase, or just as "fax")

Reply:
https://drive.google.com/file/d/13bY5eBmK6OqC6CPBnF-dolFEpaLh_fei/view?usp=sharing

@4:00 it reads "FACS" as fax, which I think is correct

Reply:
But if it reads it out as *Eff Ey See Ess* I think that's also fine

Reply:
ok thanks, we will  see what Joanna does with it :P